monitoring/uss_qualifier/scd/test_definitions/CHE/automated_test/automated_test_1.json

```diff
@@ -0,0 +1,659 @@
+{
+    "name": "Nominal Planning Test",
```

Comment:
If this is a draft of the the nominal planning test, could we put it in `CHE/astm-strategic-coordination/nominal-planning.json` to mirror [the documentation](https://github.com/interuss/dss/tree/master/interfaces/automated-testing/scd/design/astm-strategic-coordination/nominal-planning)?

Reply:
Done